src/components/NavBar/NavBar.vue

```diff
@@ -21,19 +21,25 @@
 	import NavBarSection from './NavBarSection.vue'
 	import NavBarLink from './NavBarLink.vue'
 
+
 	export default {
 		props: {
 			toc: {
 				type: Array,
 				required: true
-			}
+			},
+			
 		},
```

Comment:
These empty lines seem to be in the wrong place.. Looking at the other new lines it seems like this is meant to seperate props from data, so the new line should be after the closing bracket of the props.
```suggestion
			},
		},
		
```